src/main/resources/db/migration/V1.1__Sjp_court_load.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE IF NOT EXISTS court (
 INSERT INTO court(court_id, jurisdiction, name)
 VALUES(0, '{}', 'Single Justice Procedure (SJP)')
 ON CONFLICT (court_id) DO NOTHING;
+
+UPDATE court SET name = 'Single Justice Procedure' WHERE court_id = 0;
```

Comment:
I think we'll need to update the version of the script to 1.2, as if it's already run previously it won't update again